concepts/05 UI Components/Autocomplete/00 Getting Started with Autocomplete/00 Getting Started with Autocomplete.md

```diff
@@ -0,0 +1,11 @@
+#include uiwidgets-editors-choose-a-dropdown-editor
+
+#include tutorials-intro-installationnote
+
+Autocomplete is a text box that displays suggestions while a user types.
+
+This tutorial shows how to add an Autocomplete to the page and configure the component's core settings. As a result, you will create the following UI component:
```

Comment:
"Autocomplete" is a name of your component, therefore, you cannot use a single name with an article. Moreover, you need to have an unambiguous reference to the component later on. I mean this:

> This tutorial shows how to add an Autocomplete to the page and configure **the component**'s core settings.

It's not obvious right now that "an Autocomplete" and "the component" are referring to the very same thing.

```suggestion
This tutorial shows how to add an Autocomplete component to the page and configure the component's core settings. As a result, you will create the following UI component:
```